crates/shared/src/oneinch_api.rs

```diff
@@ -39,6 +39,43 @@ impl<const MIN: usize, const MAX: usize> Display for Amount<MIN, MAX> {
     }
 }
 
+#[derive(Debug, Clone)]
+pub struct QuoteAndSwapCommonOptions {
+    /// Contract address of a token to sell.
+    pub from_token_address: H160,
+    /// Contract address of a token to buy.
+    pub to_token_address: H160,
+    /// Amount of a token to sell, set in atoms.
```

Comment:
I think this may be a typo:
```suggestion
    /// Amount of a token to sell, set in atoms.
    pub amount: U256,
```